### PR TITLE
Back to TOC at end of each section, and small TOC for each section

### LINKED ISSUE #288: UG: Back to TOC at end of each section

### LINKED ISSUE #297: UG: TOC for each section

### PR DESCRIPTION
**Pull request by Chesterwongz**
1. closes #297 
2. closes #288

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -351,6 +364,13 @@ To find students without the `unpaid` tag and whose school is not `NYJC`, you ma
 ### Managing Lessons
 
 This section guides you on how to use the commands for managing the lessons of your students in TAB.
+
+1. [Adding a lesson: `ladd`](#adding-a-lesson-ladd)
+2. [Editing a lesson: `ledit`](#editing-a-lesson-ledit)
+3. [Deleting a lesson: `ldelete`](#deleting-a-lesson-ldelete)
+4. [Viewing a student's lessons: `view`](#viewing-lessons-view)
+5. [Viewing upcoming lessons lesson: `remind`](#viewing-upcoming-lessons-remind)
```
Comment (eeliana) [inline docs/UserGuide.md:372]:
```suggestion
5. [Viewing upcoming lessons: `remind`](#viewing-upcoming-lessons-remind)
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -472,7 +492,7 @@ Examples:
 * `find n/Betsy` followed by `ldelete 1 1` deletes the 1st lesson for the 1st student in the results 
   of the `find` command.
   
-#### Viewing lessons : `view`
+#### Viewing lessons: `view`
```
Comment (lingshanng) [inline docs/UserGuide.md:495]:
```suggestion
#### Viewing a student's lessons: `view`
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -501,12 +521,22 @@ Format: `remind`
...
+
 <div style="page-break-after: always;"></div>
 
 ### Managing Lesson Fees
 
 This section guides you on how to use the commands for managing the lesson fees of your students in TAB and the behaviour of the Fees Calculator feature of TAB.
 
+1. [Adding a Lesson's Outstanding Fees: `ladd`](#adding-a-lessons-outstanding-fees-ladd)
+2. [Editing a Lesson's Outstanding Fees: `ledit`](#editing-a-lessons-outstanding-fees-ledit)
+3. [Paying a lesson's Outstanding Fees: `paid`](#paying-a-lessons-outstanding-fees-paid)
```
Comment (lingshanng) [inline docs/UserGuide.md:538]: Only first word should be caps. Can edit the headings too?
`Behaviour of Fees Calculator` can leave it

File: docs/UserGuide.md [hunk 4]
```diff
@@ -262,6 +267,14 @@ Shows a list of all students in TAB.
 
 Format: `list`
 
+Shortcut: <kbd>F2</kbd>
```
Comment (lingshanng) [inline docs/UserGuide.md:270]: this

File: docs/UserGuide.md [hunk 5]
```diff
@@ -768,15 +856,20 @@ UI | User Interface - the means by which the user and the app interact.
...
+<br />
+
+[back to table of contents](#)
+
+<br />
+
 --------------------------------------------------------------------------------------------------------------------
 <div style="page-break-after: always;"></div>
 
 ## Command summary
```
Comment (lingshanng) [inline docs/UserGuide.md:866]:
```suggestion
## Command Summary
```

**Review by lingshanng — APPROVED** [review]
thanks!

**Pull request merged**